src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionContextExtensions.cs

```diff
@@ -110,7 +204,7 @@ private static ICollection<NotifyRangedCollectionChangedEventArgs> CreateMixedBa
                 if (previousAction.HasValue && action != previousAction.Value)
                 {
                     // Create and add event args
-                    eventArgsList.Add(new NotifyRangedCollectionChangedEventArgs(changedItems, changedItemIndices, SuspensionMode.MixedBash, previousAction.Value));
```

Comment:
@alexfdezsauco can you please revert this and the next occurence? This will not work because the other code does not allow other modes.